cmd/podman/images/sign.go

```diff
@@ -48,6 +48,10 @@ func init() {
 	flags.StringVar(&signOptions.CertDir, certDirFlagName, "", "`Pathname` of a directory containing TLS certificates and keys")
 	_ = signCommand.RegisterFlagCompletionFunc(certDirFlagName, completion.AutocompleteDefault)
 	flags.BoolVarP(&signOptions.All, "all", "a", false, "Sign all the manifests of the multi-architecture image")
+
+	authFileFlagName := "authfile"
+	flags.StringVarP(&signOptions.AuthFile, authFileFlagName, "", "", "Name of the auth file")
```

Comment:
This should use `auth.GetDefaultAuthFile()`. Similar to this: https://github.com/containers/podman/blob/95ac8f180f26d7ed4cf2945aa206a156b7a169ca/cmd/podman/images/pull.go#L107-L109